src/libs/utils.js

```diff
@@ -1,10 +1,31 @@
 import _ from 'lodash'
 import * as Config from 'src/libs/config'
 
+const subscribable = () => {
+  let value = undefined
+  let subscribers = []
+  return {
+    subscribe: fn => {
+      fn(value)
+      subscribers = _.union(subscribers, [fn])
+    },
+    unsubscribe: fn => {
+      subscribers = _.difference(subscribers, [fn])
```

Comment:
Seems like you might want to assert that the `fn` exists to prevent someone from accidentally using an anonymous function.

Reply:
Ok, will do.